docs/CANBus.md

```diff
@@ -0,0 +1,391 @@
+# CAN Bus support
```

Comment:
Note that the first sentence [here](https://github.com/lf-edge/eve/blob/master/docs/DEVICE-CONNECTIVITY.md#physical-network-ports) mentions CAN. However, since CAN is not really a network interface, it probably should not be there. Please remove it from there if you agree.

Reply:
Thanks for pointed out @milan-zededa . It's done.